package.json

```diff
@@ -5,16 +5,18 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
+    "lint": "vue-cli-service lint --no-fix",
+    "lint:fix": "vue-cli-service lint",
     "lint:test": "eslint test/**/*.test.js",
-    "test:server:up": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-up",
-    "test:server:down": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-down",
-    "test:server:clean": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-clean",
-    "test:server:restart": "yarn test:server:down; yarn test:server:clean && yarn test:server:up",
-    "test:client:up": "sh ./scripts/start.sh",
+    "lint:test:fix": "eslint --fix test/**/*.test.js",
+    "test": "tape -r esm test/**/*.test.js | tap-spec",
     "test:client:down": "sh ./scripts/stop.sh",
     "test:client:restart": "yarn test:client:down; yarn test:client:up",
-    "test": "tape -r esm test/**/*.test.js | tap-spec"
+    "test:client:up": "sh ./scripts/start.sh",
+    "test:server:clean": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-clean",
+    "test:server:down": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-down",
+    "test:server:restart": "yarn test:server:down; yarn test:server:clean && yarn test:server:up",
+    "test:server:up": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-up"
```

Comment:
These will need to be re-thought at some time. I'm happy to defer them to later, though

Reply:
Can you file an issue with a description of what needs to change? This particular change was simply the linter alphabetizing the scripts in package.json, so this wouldn't be the right time to make other, non-lint-related changes to this.

Reply:
See #73